### DISCUSSION TITLE
3D Printed Enclosure

### DISCUSSION THREAD

**Discussion by trevortrevor** [john30/ebusd#807] [General]
I have a V3.1 adapter (with ethernet). I'm hoping to design and print an case for it. Does anyone have a STEP file or similar for the completed adapter? [https://adapter.ebusd.eu/img/smd31-3deth.png](I guess like the ones used to generate these renders). I'm hoping to save some effort with the calipers.

**Comment by stuehmer** [comment]
There is some "prior art" around, maybe that helps? (I haven't tried any, yet.)
* https://www.thingiverse.com/thing:4694674
* https://www.thingiverse.com/thing:4713754
